superset/assets/javascripts/explore/stores/visTypes.js

```diff
@@ -622,6 +624,37 @@ const visTypes = {
       },
     },
   },
+  chord: {
+    label: 'Chord Diagram',
+    controlPanelSections: [
+      {
+        label: null,
+        controlSetRows: [
+          ['groupby', 'columns'],
+          ['metric'],
+          ['row_limit', 'y_axis_format'],
+        ],
+      },
+    ],
+    controlOverrides: {
```

Comment:
`controlOverrides` allows you to override the control configuration coming from `controls.jsx`. By reusing the same control across visualizations we allow continuity in the sense that the values are carried over when going from a viz type to the next. For instance as you go from one viz type to the next, it's nice to have `metrics` carried over, though you might want a slightly different label or tooltip in the different contexts, this is what the overrides are for.